config/locales/candidate_interface/other_qualification.yml

```
@@ -25,7 +25,7 @@ en:
         change_action: grade
       award_year:
         label: Year qualification was awarded
-        hint_text: For example, 1998
+        hint_text: For example, 1998. If you are studying for your qualification, indicate when you’ll acheive it.
```

Perhaps this is rare enough that the hint text is unnecessary, especially if we remove "was" from the label?
```suggestion
        hint_text: For example, 1998.
```

---

As discussed in the slack thread. Rachael decided we should leave the hint text